scripts/irods/upgrade_configuration.py

```diff
@@ -230,7 +230,12 @@ def upgrade_config_file(irods_config, path, new_version, schema_name=None):
     else:
         schema_version = schema_version_as_int(new_version['previous_version']['configuration_schema_version'])
 
-    target_schema_version = schema_version_as_int(new_version['configuration_schema_version'])
+    if 'schema_version' in new_version:
+        # Given that the schema version can be greater than 9, we have to extract
+        # all bytes following the "v" prefix in "schema_version".
```

Comment:
Just for my sanity, is this comment just explaining why L236 doesn't read like this?
```python
target_schema_version = schema_version_as_int(new_version['schema_version'][1])
```
I didn't understand what the comment about being greater than 9 was saying, so I wanted to make sure :)

Reply:
b/c numbers vs strings i think

Reply:
It is because of what @trel said.

`schema_version` is a string like `"v4"`. It could one day be something like `"v12"`. Therefore, we have to capture the bytes after "v" to form the correct integer value.

Reply:
Cool